src/components/Footer/Footer.scss

```
@@ -1,3 +1,108 @@
-* {
-  width: 100%;
+.footer {
+  background-color: #3e3e3e;
+
+  .footerContainer {
+    padding-top: 65px;
+    margin: 0 20px;
+
+    .footerTop {
+      margin-bottom: 40px;
+      display: flex;
+      justify-content: space-between;
+
+      .logoAndSlogan {
+        height: 103px;
+        width: 137px;
+
+        .footerLogo {
+          width: 120px;
+        }
+
+        .subtitle {
+          margin-top: 28px;
+          padding-top: 15px;
+          color: #9b9b9b;
+          font-size: 13px;
+
+          &:before {
+            content: '';
+            display: block;
+            margin: -20px 0 20px 0;
+            width: 30px;
+            height: 2px;
+            background-color: #6a6a6a;
+          }
+        }
+      }
+      .companyIntro {
+        line-height: 30px;
+        color: #9b9b9b;
+      }
+
+      .siteInfo {
+        line-height: 30px;
+        color: #9b9b9b;
+      }
+
+      .snsLink {
+        .fa-facebook-f {
+          border: 1px solid #9b9b9b;
+          border-radius: 50%;
+          padding: 15px 17px 15px 17px;
```

```suggestion
          padding: 15px 17px;
```
위와 같이 쓸 수 있습니다. 리뷰 드리지 않은 다른 부분도 전체적으로 확인하고 수정해주세요.